manifests/init.pp

```diff
@@ -32,15 +32,15 @@
 #
 class bolt (
   String[1] $version = 'installed',
-  Stdlib::HTTPSUrl $base_url = 'https://yum.puppet.com/',
-  String[1] $release_package = "puppet-tools-release-el-${facts['os']['release']['major']}.noarch.rpm",
-  String[1] $gpgkey = 'RPM-GPG-KEY-puppet-20250406',
-  Boolean $use_release_package = true,
+  Stdlib::HTTPSUrl $base_url = $facts['os']['family'] ? { 'Debian' => 'https://apt.puppet.com/', 'RedHat' => 'https://yum.puppet.com/', },
```

Comment:
i do not like this. not at all. but have no better idea. for my understanding this should be elaborated int the class body

Reply:
If I do it in the body, the user cannot overwrite it.

Reply:
yeah, as i say, i just dont like the style, feel free to proceed 😅